google_photos_takeout_helper/__main__.py

```diff
@@ -483,41 +515,39 @@ def fix_metadata(file: Path):
             has_nice_date = True
             return
         except FileNotFoundError:
-            print("Couldn't find json for file :/")
+            print("Couldn't find json for file ")
```

Comment:
Removed the emoji face as it looked like a messed up file url which could cause confusion

Reply:
Huh, I didn't think about this...

Reply:
I started adding `emoji` library...

Wanted to test if it works on Windoza (of course it won't)

I've spin up a vm, installed python on it...

I typed `python3`, and it started opening Windoza Store :flushed: :flushed: :flushed: 

Finally tested, and of course it doesn't work :disappointed: so i'll give up on emojis :cry:

Reply:
😢😢😢